test/unit/notify/send-emails.test.js

```diff
@@ -28,25 +28,58 @@ describe('send emails', () => {
     defaultEmailAddresses = env.notifyEmailAddresses.split(',')
 
     jest.mock('uuid', () => ({ v4: () => mockReference }))
+
+    formatEmailAddresses.mockReturnValue(
+      emailAddresses
+    )
   })
 
   afterEach(() => {
     jest.clearAllMocks()
   })
 
+  test('should call formatEmailAddress when emailAddress is a string', async () => {
+    const emailAddress = emailAddresses[0]
+    formatEmailAddresses.mockReturnValue(emailAddresses)
```

Comment:
This is already the  `mockReturnValue` in line 32, therefore we dnt need this here 

And sim. for below tests